std_msgs/Header header
float64 x
float64 y
float64 w
float64 h
int16 image_w
int16 image_h
float64 confidence